README.md

```diff
@@ -73,6 +83,34 @@ You'll need a trello account and fill in your trello credentials in the .env fil
 
 Please ensure that you populate the `.env` file with the details of names and IDs of your respective lists within Trello. 
 
+### ** Testing **
+
+The application has been built with some standard tests to ensure that the everything is fully working.  In order to use these, you need to run the following python test files 
 
+``` 
+$ python test_view_model.py
```

Comment:
I would expect `poetry run pytest` as the command to run tests. Like that it will run all test files it an find, or you can specify a folder or individual test file.